payment/controllers.go

```diff
@@ -500,28 +503,101 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 			)
 		}
 
-		creds, err := service.Datastore.GetOrderCreds(*orderID.UUID(), false)
+		var credentialType string
+		const (
+			singleUse   = "single-use"
+			timeLimited = "time-limited"
+		)
+
+		// get the order from datastore
+		order, err := service.Datastore.GetOrder(*orderID.UUID())
 		if err != nil {
-			return handlers.WrapError(err, "Error getting claim", http.StatusBadRequest)
+			return handlers.WrapError(
+				err,
+				"failed to get order", http.StatusInternalServerError)
+		}
+
+		// look through order, find out what all the order item's credential types are
+		for i, v := range order.Items {
+			if i > 0 {
+				if v.CredentialType != credentialType {
+					// all the order items on the order need the same credential type
```

Comment:
in the future we will likely need the ability to have mixed credential types on a single order

Reply:
https://github.com/brave-intl/bat-go/issues/898